docs/en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Managing Security Issues
+summary: This document highlights how Silverstripe CMS team handles security issue.
+iconBrand: github-alt
+---
+
+# Managing Security Issues
+
+This document aims to provide a high level overview of how the Silverstripe CMS team handles security issues. Only members of the Silverstripe CMS security team can perform the actions outline in this document.
+
+## Identifying/reporting security issues confidentially 
+
+This process is relevant when a potential vulnerability is reported confidentially and the Silverstripe CMS development team is in a position to prepare a patch prior to the public disclosure of the vulnerability.
+
+This process is usually started once someone [reports a security issue](Issues-and-Bugs/##security-issue).
```

Comment:
Only one hash - and I think we should revert to the auto-generated anchor

Reply:
Also I'm not sure if capital letters will work here? The actual URL is all lower case.